atable/src/components/ATableHeader.vue

```diff
@@ -1,7 +1,7 @@
 <template>
 	<thead v-if="columns.length">
 		<tr class="atable-header-row" tabindex="-1">
-			<th v-if="tableData.zeroColumn" id="header-index" />
+			<th v-if="tableData.zeroColumn" class="header-index" />
```

Comment:
@Alchez  I would love to find another selector for this or even do the same v:bind sort of trick you did elsewhere. I was looking for documentation to use `ref` as a style selector but came up empty. I think a class that isn't designed to be exposed is going to end up being leaky CSS.

Reply:
@agritheory I was able to move this back to an ID selector with the changes you made, and the zero-column width worked as expected. Since the CSS in that file is scoped, should we still look at alternatives?

Reply:
Yeah, that should be fine.